static/images/LICENSE

```diff
@@ -0,0 +1,18 @@
+Images in this directory are:
+(c) Map Tiler  (c) OpenStreetMap contributors
+
```

Comment:
...and add this from NOTICE:
See following link for data on current placeholder image:
https://www.maptiler.com/maps/#b992018d-d296-4562-9e33-2f86e28d0e2c//vector/15.78/4.49192/51.928839